src/fastoad/notebooks/02_explanation/05_FAST-OAD.ipynb

```diff
@@ -0,0 +1,441 @@
+{
```

Comment:
<blockquote>a intermediate</blockquote><p>an intermediate</p><p> </p><blockquote>Said change is the component that creates the design loop.</blockquote><p>I do not get this "Said"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Changed in dfe233f7

Reply:
<blockquote>This change is the component that creates the design loop.</blockquote><p> </p><p>--&gt; "This change <strong>concerns </strong>..." ?</p>

Reply:
<blockquote>You will notice that six IDs are listed and not one,</blockquote><p>Actually, we get more than 6, since it depends on installed plugins. But we get indeed 6 <code>tutorial.*</code> IDs.</p>

Reply:
Changed it in 77d98d3c